twisted/internet/test/test_socket.py

```diff
@@ -79,7 +81,10 @@ def test_invalidDescriptor(self):
         exc = self.assertRaises(
             socket.error,
             reactor.adoptStreamPort, fileno, socket.AF_INET, ServerFactory())
-        self.assertEqual(exc.args[0], errno.EBADF)
+        if platform.isWindows() and _PY3:
+            self.assertEqual(exc.args[0], errno.WSAENOTSOCK)
```

Comment:
Using separate `self.assertEquals` might seem redundant, but I like it because the traceback will point at back to this line and the symbolic name for this constant.